backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/badge/BpkBadge.kt

```diff
@@ -60,26 +63,38 @@ fun BpkBadge(
     Row(
         modifier = modifier
             .semantics(mergeDescendants = true) { }
-            .requiredHeight(BpkSpacing.Lg)
             .border(1.dp, type.borderColor, BadgeShape)
             .background(type.backgroundColor, BadgeShape)
-            .padding(horizontal = BpkSpacing.Md),
+            .padding(horizontal = BpkSpacing.Md, vertical = BpkSpacing.Sm),
         horizontalArrangement = Arrangement.spacedBy(BpkSpacing.Sm),
         verticalAlignment = Alignment.CenterVertically,
     ) {
         val contentColor = type.contentColor
```

Comment:
I think we need to update the text colour as well looking at the dark mode screenshots the text is quite hard to read for some